gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/gc/GcProfile.java

```diff
@@ -38,6 +38,9 @@
     class Throughput implements GcProfile {
         @Override
         public final List<String> gcJvmOpts(JavaVersion javaVersion) {
+            if (javaVersion.compareTo(JavaVersion.toVersion("11")) >= 0) {
+                return ImmutableList.of("-XX:+UseG1GC", "-XX:+UseStringDeduplication");
+            }
```

Comment:
For throughput on JDK11+ I think we still want `-XX:+UseParallelOldGC`.

Reply:
I could see potentially using G1GC & enabling string dedupe (`UseStringDeduplication` disabled by default on JDK11) on JDK11+ for `ResponseTime` but I believe some latency sensitive services saw perf regressions with G1 in certain conditions on JDK11 specifically and switched back to CMS, so should be cautious with general fleet rollout.